Documentation/Introduction/Index.rst

```diff
@@ -0,0 +1,37 @@
+﻿.. ==================================================
+.. FOR YOUR INFORMATION
+.. --------------------------------------------------
+.. -*- coding: utf-8 -*- with BOM.
+
+.. include:: ../Includes.txt
+
+
+.. _introduction:
+
+Introduction
+============
+
+
+.. _what-it-does:
+
+What does it do?
+----------------
+
+This extensions provides the possibility to integrate parts of Service BW (https://www.service-bw.de/) to your website.
+
+You can integrate the following parts:
+
+- Departments (Organisationseinheiten)
+- Circumstance (Lebenslagen)
+- Services (Leistungen)
+
+.. _screenshots:
+
+Screenshots
+-----------
+
+   I´ll add some screenshots how the frontend can look like after the TER template is ready!
+
+.. =============================
```

Comment:
Please add a note, that we are using the german method names of the server BW API in our own extension, so that's easier to extend or understand our extension.

Reply:
Added to developer corner